operators/src/upgrade/rest/service.rs

```diff
@@ -125,6 +138,176 @@ pub async fn apply_upgrade() -> Result<HttpResponse, RestError> {
     }
 }
 
+/// Upgrade data plane by controlled restart of io-engine pods
+pub async fn upgrade_data_plane() -> Result<(), Error> {
+    let pods: Api<Pod> = Api::namespaced(
+        UpgradeConfig::get_config().k8s_client().client(),
+        UpgradeConfig::get_config().namespace(),
+    );
+
+    let io_engine_listparam = ListParams::default().labels(IO_ENGINE_POD_LABEL);
+    let initial_io_engine_pod_list: ObjectList<Pod> = pods.list(&io_engine_listparam).await?;
+    for pod in initial_io_engine_pod_list.iter() {
+        // Fetch the node name on which the io-engine pod is running
+        let node_name = pod
+            .spec
+            .as_ref()
+            .ok_or_else(|| anyhow::anyhow!("pod.spec is empty"))?
+            .node_name
+            .as_ref()
+            .ok_or_else(|| anyhow::anyhow!("pod.spec.node_name is empty"))?
+            .as_str();
+
+        tracing::info!(
+            "Upgrade starting for data-plane pod: {:#?} present on node: {:#?} ",
+            pod.metadata.name,
+            node_name
+        );
```

Comment:
Use field attributes on all tracing events, eg:
```suggestion
        tracing::info!(
            pod.name = %pod.metadata.name.unwrap_or_default(),
            node.name = %node_name,
            "Upgrade starting for data-plane pod"
        );
```
This makes it more machine readable and easy to search across the traces.